dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/NetUtils.java

```diff
@@ -42,7 +42,8 @@
  */
 public class NetUtils {
 
-    private static final Pattern IP_PATTERN = Pattern.compile("\\d{1,3}(\\.\\d{1,3}){3,5}$");
+    private static final Pattern IP_PATTERN = Pattern
```

Comment:
It's better to directly use `IPAddressUtil.isIPv4LiteralAddress()`.

Reply:
ohh, i agree with u.  I always thought that `IPAddressUtil.isIPv4LiteralAddress()` used the same pattern as DS, but just checked the source code again and found that the maximum value was also judged. i will update it later. thx.
![image](https://user-images.githubusercontent.com/20518339/188320928-8fbfd2c0-0455-4be9-971e-786003faebc7.png)